src/Compilers/Server/VBCSCompilerTests/VBCSCompilerTests.csproj

```diff
@@ -15,7 +15,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
`NamedPipeServerStream.WaitForConnectionAsync(CancellationToken)` is only available in 4.6.  Made the test code significantly easier to author.